documentation/shared/attributes.adoc

```diff
@@ -87,7 +87,7 @@
 :KubernetesExternalDNS: link:https://github.com/kubernetes-incubator/external-dns[External DNS^]
 :ApacheKafkaBrokerConfig: link:https://kafka.apache.org/documentation/#brokerconfigs[Apache Kafka documentation^]
 :ApacheKafkaConnectConfig: link:https://kafka.apache.org/documentation/#connectconfigs[Apache Kafka documentation^]
-:ApacheZookeeperConfig: link:https://zookeeper.apache.org/doc/r3.5.8/zookeeperAdmin.html[ZooKeeper documentation^]
+:ApacheZookeeperConfig: link:https://zookeeper.apache.org/doc/current/[ZooKeeper documentation^]
```

Comment:
`current` here links to ZooKeeper 3.8. Kafka is using 3.6.3. I think we should point to a specific version.

Reply:
I was trying to avoid version, but updated to `https://zookeeper.apache.org/doc/r3.6.3/`